Source/WebKit/Shared/IPCTesterReceiverSwift.swift

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2025 Apple Inc. All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions
+// are met:
+// 1. Redistributions of source code must retain the above copyright
+//    notice, this list of conditions and the following disclaimer.
+// 2. Redistributions in binary form must reproduce the above copyright
+//    notice, this list of conditions and the following disclaimer in the
+//    documentation and/or other materials provided with the distribution.
+//
+// THIS SOFTWARE IS PROVIDED BY APPLE INC. AND ITS CONTRIBUTORS ``AS IS''
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+// THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+// PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL APPLE INC. OR ITS CONTRIBUTORS
+// BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
+// THE POSSIBILITY OF SUCH DAMAGE.
+
+#if ENABLE_IPC_TESTING_API && ENABLE_IPC_TESTING_SWIFT
+
+internal import WebKit_Internal
+internal import wtf
+
+// Proxy interface to test IPC activities related to receiving messages in Swift.
+class IPCTesterReceiverSwift {
+    // Optional just because of an initialization order issue. Always occupied after initialization finished.
+    var messageForwarder: RefIPCTesterReceiverSwiftMessageForwarder?
+
+    init() {
+        self.messageForwarder = WebKit.IPCTesterReceiverSwiftMessageForwarder.create(target: self)
+    }
+
+    func getMessageReceiver() -> RefIPCTesterReceiverSwiftMessageForwarder {
+        guard let messageForwarder = self.messageForwarder else {
+            fatalError("Unreachable - guaranteed to exist")
+        }
+        return messageForwarder
+    }
+
+    func asyncMessage(data: UInt32, completionHandler: IntCompletionHandler) {
```

Comment:
can this be named something more realistic, like `computeSuccessiveValue` or something like that

Reply:
I'm trying to match the names and functionality of the pre-existing identical C++ tester, so I'll keep it as-is.